### PR TITLE
feat: create `complianceCheckStatus` widget

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
Last widget markup!!!
스크린샷 2023-06-01 오후 5 21 01

### CODE REVIEW

File: apps/web/src/services/dashboards/widgets/compliance-check-status/ComplianceCheckStatusWidget.vue [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+                        key: 'limit',
+                        operator: 'sum',
+                    },
+                    budget_count: {
+                        operator: 'count',
+                    },
+                },
+                ...apiQueryHelper.data,
+            },
+        });
```
Comment (WANZARGEN) [inline apps/web/src/services/dashboards/widgets/compliance-check-status/ComplianceCheckStatusWidget.vue:199]: praise: 👍

**Review by WANZARGEN — APPROVED** [review]
LGTM overally~ I left a small comment!

File: apps/web/src/services/dashboards/widgets/compliance-check-status/ComplianceCheckStatusWidget.vue [hunk 2]
```diff
@@ -0,0 +1,368 @@
...
+        toggleKey: 'none',
+        forceInactive: true,
+        templateField: 'pieSettings',
+        strokeOpacity: 0,
+    });
+
+    // TODO: replace with real data
+    setPieLabelText(chart, { text: '[fontSize:16px]Compliance score[/]:\n[fontSize:32px]45[/]' });
+    innerSeries.data.setAll(innerChartData);
+};
```
Comment (sulmoJ) [inline apps/web/src/services/dashboards/widgets/compliance-check-status/ComplianceCheckStatusWidget.vue:260]: Praise: Super Cooool

**Review by sulmoJ — APPROVED** [review]
Thank you, chart master!

**Pull request merged**